src/dodal/devices/watsonmarlow323.py

```diff
@@ -0,0 +1,53 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_rw
+
+class WatsonMarlow323Enable(str, Enum):
+     DISABLED = "Disabled"
+     ENABLED = "Enabled"
+
+class WatsonMarlow323Direction(str, Enum):
+     CW = "CW"
+     CCW = "CCW"
```

Comment:
Nit: More meaningful names would be good here, we need to retain the cryptic acronyms on the epics side, but the ophyd side can have long, descriptive names e.g.

```suggestion
class WatsonMarlow323Direction(str, Enum):
     CLOCKWISE = "CW"
     COUNTER_CLOCKWISE = "CCW"
```

Reply:
Updated names in 417aaa12897e1aad61c73db30d4f3d6e00974e29